pandera/pandera.py

```diff
@@ -229,22 +250,27 @@ def prepare_series_input(self, series, dataframe):
 
         return self._format_input(groupby_obj, self.groups)
 
-    def prepare_dataframe_input(self, dataframe):
+    def prepare_dataframe_input(self,
+                                dataframe: pd.DataFrame):
         """Prepare input for DataFrameSchema check."""
         if self.groupby is None:
             return dataframe
         else:
             groupby_obj = dataframe.groupby(self.groupby)
         return self._format_input(groupby_obj, self.groups)
 
-    def _vectorized_check(self, parent_schema, check_index, check_obj):
+    def _vectorized_check(self,
+                          parent_schema: Union[DataFrameSchema, SeriesSchemaBase],
```

Comment:
unfortunately these classes are defined below and is causing the issue on CI: https://travis-ci.org/pandera-dev/pandera/jobs/574945468#L830

I think we can forgo type annotations that involve pandera classes until code has been modularized

Reply:
Will resolve after modularisation